### ISSUE TITLE
How do I reproduce an ALPHABETICAL_EXPANDED output with skos play cli?

### ISSUE THREAD

**Issue by eonm-abes** [sparna-git/skos-play#82]
Hi @tfrancart ,

Is it possible to produce an ALPHABETICAL_EXPANDED output with skos play cli ?

Can I customize HTML output ?

Best Regards

**Comment by tfrancart** [comment]
> Is it possible to produce an ALPHABETICAL_EXPANDED output with skos play cli?

Yes, I think this is what the "alphabetical" command does by default. See, in the controller of the web UI:

![image](https://github.com/sparna-git/skos-play/assets/2728945/737ad3b8-1775-42cb-a2c4-33c699f8a902)

and in the skos-play CLI "alphabetical" command : 

![image](https://github.com/sparna-git/skos-play/assets/2728945/324c13ca-507c-42a7-ab26-1b633c6678d1)

> Can I customize HTML output?

The only customisation you can do is override the CSS styles with your own. You cannot customise the HTML structure.
